### PR TITLE
fix cassandra code checkstyle

### PR DESCRIPTION
**Pull request by z7658329**
fix cassandra code checkstyle

### CODE REVIEW

**Comment by z7658329** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by z7658329** [comment]
check result:
image

**Review by javeme — APPROVED** [review]

**Review by imbajin — APPROVED** [review]

**Review by coderzc — APPROVED** [review]

File: hugegraph-cassandra/src/main/java/com/baidu/hugegraph/backend/store/cassandra/CassandraTable.java [hunk 1]
```diff
@@ -107,12 +107,12 @@ public Number queryNumber(CassandraSessionPool.Session session,
             statement.setReadTimeoutMillis(timeout * 1000);
             return session.query(statement);
         }, (q, rs) -> {
-            Row row = rs.one();
-            if (row == null) {
-                return IteratorUtils.of(aggregate.defaultValue());
-            }
-            return IteratorUtils.of(row.getLong(0));
-        });
+                Row row = rs.one();
```
Comment (javeme) [inline hugegraph-cassandra/src/main/java/com/baidu/hugegraph/backend/store/cassandra/CassandraTable.java:110]: can we keep the origin indentation style?

**Review by javeme — APPROVED** [review]

File: hugegraph-cassandra/src/main/java/com/baidu/hugegraph/backend/store/cassandra/CassandraTable.java [hunk 2]
```diff
@@ -107,12 +107,12 @@ public Number queryNumber(CassandraSessionPool.Session session,
...
+                if (row == null) {
+                    return IteratorUtils.of(aggregate.defaultValue());
+                }
+                return IteratorUtils.of(row.getLong(0));
+            });
```
Comment (imbajin) [inline hugegraph-cassandra/src/main/java/com/baidu/hugegraph/backend/store/cassandra/CassandraTable.java:115]: how could it add 8 space in IDE's behavior? seems it's confuse

**Pull request closed without merging**